fusion_docs/faq.md

```diff
@@ -48,7 +48,7 @@ If you didn’t notice any performance improvement with Fusion, the bottleneck m
 
 Yes. Add the Fusion version's config URL using the `containerConfigUrl` option in the Fusion block of your Nextflow configuration (replace `v2.4.2` with the version of your choice):
 
-```groovy
+```nextflow
```

Comment:
In the nextflow docs, I only use the `nextflow` highlighter for script code blocks and I still use `groovy` for config code. since the config doesn't have any special keywords and using `nextflow` here will only cause things to be incorrectly highlighted (like the `process` scope)

Reply:
Uff, ok. Then that means I can't just use `sed` to find + replace all occurrences.. 😞  Fair enough.

Reply:
Amazing. There wasn't one single Nextflow script code block in the entire Seqera docs codebase (Fusion + Platform). I went through one by one and everything was config 😆